.changeset/curly-moons-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+'create-svelte': major
+---
+
+Switch default ESLint plugin from `eslint-plugin-svelte3` to `eslint-plugin-svelte`
```

Comment:
should probably use a prefix to match the rest of the changelog: https://github.com/sveltejs/kit/blob/master/packages/create-svelte/CHANGELOG.md
```suggestion
feat: switch default ESLint plugin from `eslint-plugin-svelte3` to `eslint-plugin-svelte`
```